bin/generate_kong_config

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+. bin/util
+
+announce "Bundling OpenAPI spec"
+bin/bundle_spec
+
+# a Kong declarative configuration requires a defined target server
+# using http://host.docker.interal allows a containerized Kong Gateway to target
+# a local Conjur instance
+echo "servers:
+  - url: http://host.docker.internal:80" >> spec.yml
+
+announce "Generating Kong declarative configuration yaml"
+docker run --rm -it \
+    -v ${PWD}:/opt/openapi \
+    node:latest \
+    bash -c "
+        npm i -g insomnia-inso \
```

Comment:
I might add a note here as to what insomnia is/what you are doing with it

Reply:
Added this information in `examples/kong/README.md`